deploy/verify_polyscan.js

```diff
@@ -0,0 +1,23 @@
+const { ethers } = require('hardhat');
```

Comment:
I think this file should be moved to `/scripts` and lets use the default naming patter for js files `verify-polyscan.js`

Reply:
local_deploy.js and mumbai_deploy.js both use underscores

Reply:
moved the file to scripts.